components/finder.rst

```diff
@@ -223,6 +223,11 @@ The :method:`Symfony\\Component\\Finder\\Finder::notPath` method excludes files
 
     $finder->notPath('other/dir');
 
+The method :method:`Symfony\\Component\\Finder\\Finder::path` matches both directories and files. For example, the following will match ``foo/bar.yml`` but also ``foo.yml``::
```

Comment:
And can you please wrap the paragraph after the first line that crosses the 72nd character?